core/src/main/scala/cats/data/NonEmptyList.scala

```diff
@@ -82,6 +82,9 @@ final case class NonEmptyList[+A](head: A, tail: List[A]) {
   def ::[AA >: A](a: AA): NonEmptyList[AA] =
     NonEmptyList(a, head :: tail)
 
+  def :::[AA >: A](other: NonEmptyList[AA]): NonEmptyList[AA] =
```

Comment:
Actually to maintain symmetry, can we add `++:` to `NonEmptyVector` as well?

Reply:
Sure :)